photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -131,6 +131,48 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
     assertTrue(AvroUtils.modelContainsIntercept(sc, bestFixedEffectModelPath))
   }
 
```

Comment:
It would be good idea to add integTest for hyper-parameter tuning for randomEffect as well.

Reply:
Updated to have the hyper-parameter tuning test run on mixed effects and compare the tuned models against the best model.